lib/compute/disk.js

```diff
@@ -57,13 +70,92 @@ var util = require('../common/util.js');
  * var disk = zone.disk('disk1');
  */
 function Disk(zone, name) {
-  this.zone = zone;
+  var methods = {
+    /**
+     * Create a persistent disk.
+     *
+     * @param {object} config - See {module:compute/zone#createDisk}.
+     *
+     * @example
+     * var config = {
+     *   // ...
```

Comment:
Yeah, they go to `compute/zone#createDisk`, which is linked as the `config` parameter.